packages/page-bounties/src/helpers/extendedStatuses.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2017-2021 @polkadot/app-bounties authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { TFunction } from 'i18next';
+import type { DeriveCollectiveProposal } from '@polkadot/api-derive/types';
+import type { BountyStatus } from '@polkadot/types/interfaces';
+
+type BountyVotingStatus = 'Active' | 'Approved' | 'CuratorProposed' | 'Funded' | 'PendingPayout' | 'Proposed';
+type BountyVotingStatuses = { [status in BountyVotingStatus]: string[] };
+
+const validProposalNames: BountyVotingStatuses = {
+  Active: ['closeBounty', 'unassignCurator'],
+  Approved: [],
+  CuratorProposed: ['unassignCurator', 'closeBounty'],
+  Funded: ['proposeCurator', 'closeBounty'],
+  PendingPayout: ['unassignCurator'],
+  Proposed: ['approveBounty', 'closeBounty']
+};
+
+function votingDescription (proposalName: string | undefined, t: TFunction): string | null {
+  switch (proposalName) {
+    case ('approveBounty'):
+      return t('Approval under voting');
```

Comment:
These are problematic - becuase it is outside of a `useTranslation`, it doesn't get included in the `en/app-bounties.json`, which in turn means it won't appear for translation at all.